stubs/cachetools/@tests/stubtest_allowlist.txt

```diff
@@ -1,7 +1,22 @@
 cachetools.Cache.get
-cachetools.cache.Cache.get
 
-# it looks like cachetools stubs are really out of date, so none of the modules
-# referenced exist anymore
-cachetools\..*\.__(get|set|del)item__  # has extra argument
-cachetools\..*TTLCache.__repr__
+# stubs omit defaulted arguments that are meant to be optimizations, not provided by user
```

Comment:
Does it hurt that much to include the weird extra arguments in these signatures? I worry that if we allowlist all these functions, stubtest won't warn us if there are any other problems with any of them

Reply:
And after thinking about this a bit more: Including the weird extra arguments also would mean including a new method definition for each of the lines I whitelisted. That would clutter the stubs and not help users in any way. CI checks should catch problems, not force you to do something weird and unnecessary IMO.

Reply:
Okay, sounds good! Will try to do a full review of the rest of the PR soon.

Reply:
Side point: I'm amazed a library that's concerned with such micro-optimisations is also using `functools.total_ordering` in the same module.